resolwe_bio/processes/import_data/metadata.py

```diff
@@ -60,8 +62,28 @@ def change_suffix(path):
     return new_path
 
 
-class UploadOrangeMetadata(Process):
-    """Upload metadata table in Orange format.
+def convert_to_data_frame(path):
+    """Convert the uploaded file to Pandas data frame."""
+    if path.endswith("xls"):
+        df = pd.read_excel(path, engine="xlrd")
+    elif path.endswith("xlsx"):
+        df = pd.read_excel(path, engine="openpyxl")
+    elif any(path.endswith(ext) for ext in ["tab", "tsv"]):
+        df = pd.read_csv(path, sep="\t")
+    elif path.endswith("csv"):
+        df = pd.read_csv(path)
+    else:
+        df = pd.DataFrame()
+    return df
+
+
+class UploadOrangeMetadataOneToOne(Process):
+    """Upload one-to-one metadata table in Orange format.
+
+    The uploaded metadata table represents one-to-one (1:1) relation to
+    samples in the working collection. Metadata table must contain a column
+    with one of the following headers: "mS#Sample ID", "mS#Sample name" or
+    "mS#Sample slug".
 
     Orange can read files in native tab-delimited format, or can load
```

Comment:
If we decide to remove the `mS#` prefixes, this description is no longer necessary.